jpos/src/test/java/org/jpos/space/JDBMSpaceTestCase.java

```diff
@@ -155,6 +155,36 @@ public void testPush() {
         assertNull (sp.rdp ("PUSH"));
     }
     @Test
+    public void testOutExpire() {
+        sp.out ("PUSH", "ONE", 1000L);
```

Comment:
Not a big deal, key name can be anything, but "PUSH" in the `testOutExpire` is probably confusing to a new user. Perhaps you want to use "OUT" as the key, or any other key.

Reply:
right, will change